### ISSUE TITLE
No scrollbar in settings

### ISSUE THREAD

**Issue by LeafyLappa** [ani-memes/AMII#127]
Hi!

This one is quite weird but I can't scroll the plugin settings. Scrollbar is missing and I have to resize the settings window, otherwise things are inaccessible, being outside its viewport. I only have this issue with this particular plugin.

![hBwbnbPLGR](https://user-images.githubusercontent.com/31704619/133249902-dcb06e92-deb3-41e9-bc9a-b2a7ecb63cec.gif)

**Comment by Dragon-0609** [comment]
Please, write IDE and plugin info.

**Comment by Unthrottled** [comment]
> Please, write information about the IDE and plugin. 
> Such as which IDE, which version; which version of the plugin

You can find this information in `Help` -> `About`, you should be able to copy and paste the information here.

I am fairly confident that this an issue with `General` Settings, right? I accidentally broke the fixed sizing of the gif in `v0.11.0`, that should be fixed in [v0.12.0](https://github.com/ani-memes/AMII/releases/tag/v0.12.0) if you wanted to manually install that version. However, there is not anything below the gif :slightly_smiling_face: 

![Peek 2021-09-14 17-32](https://user-images.githubusercontent.com/15972415/133342921-ba91c07d-2af8-47af-a0c8-6ab1e42739fd.gif)

**Comment by LeafyLappa** [comment]
Thank you for the quick response.

IDE from the gif in my first post:
```
IntelliJ IDEA 2021.2.1 (Community Edition)
Build #IC-212.5080.55, built on August 24, 2021
Runtime version: 11.0.11+9-b1504.16 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 2000M
Cores: 4
Registry: ide.balloon.shadow.size=0
Non-Bundled Plugins: some.awesome (1.14), ski.chrzanow.foldableprojectview (1.1.1), com.c5inco.modifiers (0.1.9), mobi.hsz.idea.gitignore (4.2.0), io.acari.DDLCTheme (18.0.0), org.jetbrains.kotlin (212-1.5.30-release-409-IJ4638.7), org.jetbrains.compose.desktop.ide (1.0.0-alpha4-build344), kvision.project.wizard (0.3.0), SequenceDiagram (2.1.2), com.jetbrains.edu (2021.8-2021.2-298), io.unthrottled.amii (0.11.0)
Kotlin: 212-1.5.30-release-409-IJ4638.7
```

You can see io.unthrottled.amii (0.11.0) in the list of non-bundled plugins.

I've had this problem across I believe 4 IDEA updates now, and it's also present in Android Studio.

> I am fairly confident that this an issue with General Settings, right?
> However, there is not anything below the gif 🙂

Oh but there's a thank you message there! Nevertheless settings on the Events tab don't fit either:

![CFIeNEZYoE](https://user-images.githubusercontent.com/31704619/133398037-931b1fd4-9e57-49ab-8715-d7f82ead487c.gif)

Somehow scrollbar is there now (I don't think I did anything to prompt this) but as you can see it doesn't work right.

**Comment by LeafyLappa** [comment]
Tried opening settings one more time (did exactly nothing else) and scrollbar is missing.

**Comment by Dragon-0609** [comment]
@Unthrottled , after this issue I checked in my IDE, and found that @LeafyLappa's problem is also in my IDE. I have updated plugin, in v12 the the problem is solved.

**Comment by Unthrottled** [comment]
Thank you for taking the time to submit this issue. I'm planning on publishing this release to the plugin marketplace after I fix #123
